src/Util.ts

```diff
@@ -16,6 +18,17 @@ const parseErrorFromFailedResponse = (failureResponse: Response) =>
         .errorMap(() => new TspServiceException(TenantSecurityErrorCode.UNKNOWN_ERROR, "Unknown response from Tenant Security Proxy", failureResponse.status))
         .flatMap((errorResp: ApiErrorResponse) => Future.reject(TenantSecurityExceptionUtils.from(errorResp.code, errorResp.message, failureResponse.status)));
 
+const httpAgent = new http.Agent({keepAlive: true});
+const httpsAgent = new https.Agent({keepAlive: true});
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const agentSelector = function (_parsedURL: any) {
```

Comment:
Should we comment on why this is needed?

Reply:
Yep, added